util/uvmdvgen/sim.core.tpl

```diff
@@ -18,9 +18,22 @@ filesets:
     file_type: systemVerilogSource
 
 targets:
-  sim:
+  sim: &sim_target
     toplevel: tb
     filesets:
       - files_rtl
       - files_dv
     default_tool: vcs
+
+  lint:
```

Comment:
I feel we can add a TODO here to remind people to add the new IP in hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson

Reply:
That is a good point! Thanks Weicai, I will add it to the template.

Reply:
Lets add a V1 checklist item to force people to add their block to DV lint.

Reply:
Sure, I can add a commit for that.

Reply:
Created PR https://github.com/lowRISC/opentitan/pull/4059 to add the lint requirement to V1 checklist.